client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -103,14 +91,8 @@ class InlineHelpRichResult extends Component {
 	}
 }
 
-const mapStateToProps = ( state, { result } ) => ( {
+const mapStateToProps = ( state ) => ( {
 	searchQuery: getSearchQuery( state ),
-	type: get( result, RESULT_TYPE, RESULT_ARTICLE ),
-	title: get( result, RESULT_TITLE ),
-	link: get( result, RESULT_LINK ),
-	description: get( result, RESULT_DESCRIPTION ),
-	tour: get( result, RESULT_TOUR ),
-	postId: get( result, 'post_id' ),
```

Comment:
This removal is weird because these props continue to be used in the component. They could be replaced with references to `this.props.result.type` etc but you didn't do that.

Update: now I see the props are passed from above, see a comment below.